app/layout/scenarios/reports/map/component.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useCallback, useEffect, useState } from 'react';
+
+import { useSelector } from 'react-redux';
+
+import { useRouter } from 'next/router';
+
+import { getScenarioEditSlice } from 'store/slices/scenarios/edit';
+
+import PluginMapboxGl from '@vizzuality/layer-manager-plugin-mapboxgl';
+import { LayerManager, Layer } from '@vizzuality/layer-manager-react';
+
+import { useAccessToken } from 'hooks/auth';
+import {
+  usePUGridLayer,
+} from 'hooks/map';
+import { useProject } from 'hooks/projects';
+import { useScenario } from 'hooks/scenarios';
+import { useBestSolution } from 'hooks/solutions';
+
+import Loading from 'components/loading';
+import Map from 'components/map';
+
+export interface ScenariosReportMapProps {
+}
+
+export const ScenariosReportMap: React.FC<ScenariosReportMapProps> = () => {
+  const [mapInteractive, setMapInteractive] = useState(false);
+
+  const accessToken = useAccessToken();
+
+  const { query } = useRouter();
+
+  const { pid, sid } = query;
+
+  getScenarioEditSlice(sid);
```

Comment:
You don't need to load redux, we can set (if we need them) the values from redux as constants.